etc/kayobe/ansible/filter_plugins/filters.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+from ansible.errors import AnsibleFilterError
+from ansible.plugins.filter.core import to_bool
+
+class FilterModule(object):
+    def filters(self):
+        return {
+            'select_enabled': self.select_enabled
+        }
+
+    # Takes a list of dict
+    # Returns a list of values of the key 'key' where the key 'enabled' is True
+    # If key is a list, flatten into single result list
+    def select_enabled(self, items, key='items'):
```

Comment:
Would it be better to use a Python [type signature](https://docs.python.org/3/library/typing.html) or even a [docstring](https://peps.python.org/pep-0257/) here instead of a comment?